### PR TITLE
Creating description for StorageResource

### PR DESCRIPTION
**Pull request by VanDavv**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -374,6 +374,16 @@ If anything fails, a suitable error code with description will be returned.
             "message":"Reset password failed."
         }
         
+# Group Files
```
Comment (mlotysz) [inline apiary.apib]: It's misleading. It's not about files. It's about ORC text from image and text files...

File: apiary.apib [hunk 2]
```diff
@@ -374,6 +374,16 @@ If anything fails, a suitable error code with description will be returned.
...
+
+## Files [/users/{userId}/storage]
```
Comment (mlotysz) [inline apiary.apib]: Missing description. What is this for and basic usage flow from client standpoint.

File: apiary.apib [hunk 3]
```diff
@@ -374,6 +374,16 @@ If anything fails, a suitable error code with description will be returned.
...
+
+### Flashcards from Image [POST]
+
++ Request (multipart/form-data)
```
Comment (mlotysz) [inline apiary.apib]: Missing part descriptions and names. Multi parts what part should client send, what names? One image or many? I'll confused.

File: apiary.apib [hunk 4]
```diff
@@ -374,16 +374,53 @@ If anything fails, a suitable error code with description will be returned.
...
+        -----BOUNDARY
 
 + Response 201 (application/json)
 
+# Group CV
+
+## Resources for CV server [/storage/{userId}/{fileId}]
+
+### Get file[GET]
+This resource returns file from storage using userId and fileId
```
Comment (mlotysz) [inline apiary.apib]: How to get `fileId`? Is fileId returned by `POST /decks/cv`? Should be documented there.

Comment (VanDavv) [inline apiary.apib]: They get prepared URL with fileId included

Comment (mlotysz) [inline apiary.apib]: Right. I forgot, that we are in CV super specialized flow.

**Comment by mlotysz** [comment]
Thanks!

**Pull request merged**